Project.toml

```diff
@@ -16,6 +17,7 @@ UnsafeArrays = "c4a57d5a-5b31-53a6-b365-19f8c011fbd6"
 Adapt = "1, 2, 3.3"
 FFTW = "1.2"
 LRUCache = "1"
+QuantumInterface = "0.1.0"
```

Comment:
Why not "0.1"?

Reply:
I think it means the same. `0.1.x` implies `[0.1.x, 0.2)` and `0.1` implies `[0.1.0, 0.2)` which are the same for x=0. But maybe I misunderstood https://pkgdocs.julialang.org/v1/compatibility/

Reply:
Oh, my mistake. Thanks for clarifying.